blobstore/blobnode/base/flow/flow_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 The CubeFS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package flow
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestDiskViewer(t *testing.T) {
+	ioFlowStat, err := NewIOFlowStat("default", false)
+	require.NoError(t, err)
+	ioview := NewDiskViewer(ioFlowStat)
+	statData := ioview.WriteStat()
```

Comment:
Add more testing case data.
Test with actual concurrent reads and writes on the disk viewer.

Reply:
What data to write for concurrency testing

Reply:
<img width="942" alt="image" src="https://github.com/cubefs/cubefs/assets/18587688/df3b0626-9057-42de-988d-39a1e4c94a9a">
iostat qos already has a test program, flow only displays data data, method calls do not need test